backend/oasst_backend/prompt_repository.py

```diff
@@ -291,6 +292,26 @@ def store_ranking(self, ranking: protocol_schema.MessageRanking) -> Tuple[Messag
 
         return reaction, task
 
+    def insert_toxicity(self, message_id: UUID, model: str, toxicity) -> MessageToxicity:
+        """Save the toxicity score of a new message in the database.
+        Args:
+            message_id (UUID): the identifier of the message we want to save its toxicity score
+            model (str): the model used for creating the toxicity score
+            toxicity (float): the values obtained from the message & model
```

Comment:
the doc string (float) type does not match how it is used below in line 308, e.g. it is probably a `dict[str, Any]`?